ui/app/routes/vault/cluster/clients/counts.ts

```diff
@@ -59,28 +52,19 @@ export default class ClientsCountsRoute extends Route {
     return { activity, activityError };
   }
 
-  async getActivatedFeatures() {
-    try {
-      const resp: ActivationFlagsResponse = await this.store
-        .adapterFor('application')
-        .ajax('/v1/sys/activation-flags', 'GET', { unauthenticated: true, namespace: null });
-      return resp.data?.activated;
-    } catch (error) {
-      if (DEBUG) console.error(error); // eslint-disable-line no-console
-      return [];
-    }
-  }
+  showSecretsSync(activity: ClientsActivityModel | undefined): boolean {
+    // if there is a history of sync clients, show it
```

Comment:
technically this isn't just historical data since the end time we initially query is the current month. definitely semantics, but just wanted to point that out! I might suggest rewording to something like `if there is any sync client data, show it`